int16[] pose_ids
---
geometry_msgs/PoseWithCovariance[] pose_estimates